champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/Deflection.java

```diff
@@ -46,13 +46,11 @@ public String getName() {
     public String[] getDescription(int level) {
 
         return new String[]{
-                "Prepare to deflect incoming attacks",
-                "You gain <val>1</val> charge every <val>" + timeBetweenCharges + "</val> seconds.",
+                "You gain <stat>1</stat> charge every <stat>" + timeBetweenCharges + "</stat> seconds.",
                 "You can store a maximum of <val>" + (level) + "</val> charges",
                 "",
                 "When attacked, the damage you take is",
-                "reduced by the number of your charges",
-                "and your charges are reset to 0."
```

Comment:
Why was this removed?

Reply:
redundant information that is implied by the rest of the skill description